<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogRLC</class>
 <widget class="QDialog" name="dialogRLC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RLC Specification</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="5" column="3" colspan="3">
    <widget class="QComboBox" name="comboBoxCoaxTypes"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Transmission Line Connection</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBoxRLC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>RLC</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonParallel">
        <property name="text">
         <string>Parallel</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButtonSeries">
        <property name="text">
         <string>Series</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEditSelR"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEditSelL"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_QL"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEditSelC"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="lineEdit_QC"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_R">
        <property name="text">
         <string>Resistance (ohms)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_L">
        <property name="text">
         <string>Inductance (henries)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_QL">
        <property name="text">
         <string>QL=</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_QC">
        <property name="text">
         <string>QC=</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_C">
        <property name="text">
         <string>Capacitance (farads)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="QLabel" name="A4">
     <property name="text">
      <string>line is specified.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="QLabel" name="A3">
     <property name="text">
      <string>For the series fixture the RLC values are ignored if a transmission</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="A1">
     <property name="text">
      <string>Specify type and length of transmission line, if any, and a combined</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QLabel" name="A2">
     <property name="text">
      <string>Resistor, Inductor and Capacitor as line termination.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="3">
    <widget class="QGroupBox" name="groupBoxFix">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Fixture Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>R0 (ohms)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_FixR0"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButton_SeriesFix">
        <property name="text">
         <string>Series</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButton_ShuntFix">
        <property name="text">
         <string>Shunt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox_R0">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>VF</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>K1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>K2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_VF">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_K1">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_R0">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEdit_Len">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="lineEdit_K2">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>R0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Len (ft)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="6">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
